packages/pages/src/vite-plugin/build/closeBundle/closeBundle.ts

```diff
@@ -46,7 +49,7 @@ export default (projectStructure: ProjectStructure) => {
 
     finisher = logger.timedLog({ startLog: "Writing manifest.json" });
     try {
-      await generateManifestFile(templateModules, projectStructure);
```

Comment:
Asking mostly out of curiosity but I thought typescript would have complained about this before?

Reply:
I didn't get an error, but I saw a white dotted underline with a message `'await' has no effect on the type of this expression.ts(80007)`.

Reply:
just saw your edit, yea adding an eslint rule would be good (maybe this one: https://github.com/typescript-eslint/typescript-eslint/blob/main/packages/eslint-plugin/docs/rules/await-thenable.md)

Reply:
added to the doc for capturing pagesjs repo changes!